models/things.js

```diff
@@ -116,7 +116,7 @@ var Things = {
 
      // Notify each open websocket of the new Thing
      this.websockets.forEach(function(socket) {
-       socket.send(JSON.stringify(thing.getDescription()));
+       socket.send(JSON.stringify(thing));
```

Comment:
Thanks for the clarification! I think I'll go with the first approach. I do think I've found a bug :)